data-serving/data-service/schemas/cases.indexes.json

```diff
@@ -0,0 +1,45 @@
+[
+    {
+        "name": "textIdx",
+        "key": {
+            "notes": "text",
+            "revisionMetadata.creationMetadata.curator": "text",
+            "demographics.occupation": "text",
+            "demographics.nationalities": "text",
+            "demographics.ethnicity": "text",
+            "location.country": "text",
+            "location.administrativeAreaLevel1": "text",
+            "location.administrativeAreaLevel2": "text",
+            "location.administrativeAreaLevel3": "text",
+            "location.place": "text",
+            "location.name": "text",
+            "pathogen.name": "text",
+            "caseReference.sourceUrl": "text"
+        }
+    },
+    {
+        "name": "casesCaseRefIdx",
+        "key": {
+            "caseReference.sourceId": -1,
+            "caseReference.sourceEntryId": -1
+        },
+        "unique": true,
+        "partialFilterExpression": {
+            "caseReference.sourceEntryId": {
+                "$exists": true
+            }
+        }
+    },
+    {
+        "name": "eventsArrayIdx",
+        "key": {
+            "events": -1
```

Comment:
Do we want the index on the whole event document, or a field (like name)? I'm not sure what the charts query is, apologies.

Reply:
oops you're totally right, thanks!